src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -94,6 +96,12 @@ export class UserListComponent implements OnInit {
         prop: 'roles',
         flexGrow: 1,
         cellTemplate: this.userRolesTpl
+      },
+      {
+        name: this.i18n('Enabled'),
+        prop: 'enabled',
+        flexGrow: 1,
+        cellTemplate: this.userEnabledTpl
```

Comment:
Plesae use the ``checkIconTpl`` from the table component or

``
pipe: this.booleanTextPipe
``

in the table column definition. It is not necessary to define a new template.